### ISSUE TITLE
Encountered an error while loading model file 

### ISSUE THREAD

**Issue by RizanKhan837** [ricky0123/vad#169]
keep geeting this error while i run my code 
real-time-vad.ts:238 Encountered an error while loading model file https://cdn.jsdelivr.net/npm/@ricky0123/vad-web@0.0.20/dist/silero_vad_v5.onnx

here's how i used it.
```
  const vad = useMicVAD({
    model: "v5",
    startOnLoad: true,
    onSpeechEnd: async (audio) => {
      // Prevent multiple simultaneous processing
      if (isProcessing) return;

      setIsProcessing(true);

      try {
        // Convert audio to base64 for storage or transmission if needed
        const wavBuffer = utils.encodeWAV(audio);
        const base64 = utils.arrayBufferToBase64(wavBuffer);
        const audioUrl = `data:audio/wav;base64,${base64}`;

        // Add audio to list for playback
        setAudioList((old) => [audioUrl, ...old]);

        // Convert audio blob for transcription
        const audioBlob = new Blob([wavBuffer], { type: "audio/wav" });

        // Transcribe audio
        const transcription = await STTService.transcribeAudio(audioBlob);

        if (transcription) {
          // Process transcription with local AI model
          const response = await processWithLocalAI(transcription);

          // Synthesize speech response
          if (response) {
            await TTSService.synthesizeSpeech(response);
          }
        }
      } catch (error) {
        console.error("Voice assistant error:", error);
      } finally {
        setIsProcessing(false);
      }
    },
  });
  ```
  
  here is my webpack config
  ```
  const CopyPlugin = require("copy-webpack-plugin");

module.exports = {
  entry: "./src/index.jsx",
  module: {
    rules: [
      {
        test: /\.m?jsx?$/,
        exclude: /node_modules/,
        use: {
          loader: "babel-loader",
          options: {
            presets: [["@babel/preset-react"]],
          },
        },
      },
    ],
  },
  plugins: [
    new CopyPlugin({
      patterns: [
        {
          from: "node_modules/@ricky0123/vad-web/dist/vad.worklet.bundle.min.js",
          to: "src/components/VoiceAssistant.js",
        },
        {
          from: "node_modules/@ricky0123/vad-web/dist/*.onnx",
          to: "src/components/VoiceAssistant.js",
        },
        {
          from: "node_modules/onnxruntime-web/dist/*.wasm",
          to: "src/components/VoiceAssistant.js",
        },
        { from: "src/index.html", to: "[name][ext]" },
      ],
    }),
  ],
  output: {
    clean: true,
  },
};

  ```

**Comment by zhenxy0503** [comment]
me too

**Comment by ricky0123** [comment]
Hi, can you look in the network tab of browser dev tools and see where it is trying to load the vad from? Is it getting a 404?

**Comment by zhenxy0503** [comment]
> Hi, can you look in the network tab of browser dev tools and see where it is trying to load the vad from? Is it getting a 404?

I'll open it directly https://cdn.jsdelivr.net/npm/ @ricky0123/ vad-web@0.0.20 /The website dist/selero-vad-v5. onnx can be accessed by browsers, but the page prompts Encountered an error while loading model file https://cdn.jsdelivr.net/npm/ @ricky0123/ vad-web@0.0.20 /Dist/silero-vad-v5,onnx, I found that these files are included in the dependency package @ ricky0123/vad-web, but I tried to load them locally without success

**Comment by 77hole** [comment]
> Hi, can you look in the network tab of browser dev tools and see where it is trying to load the vad from? Is it getting a 404?

yes，i meet it too,are you solving

**Comment by 77hole** [comment]
> Hi, can you look in the network tab of browser dev tools and see where it is trying to load the vad from? Is it getting a 404?

yes, i meet it too,how can i solve it?

**Comment by SafeeUrRehman65** [comment]
Hey @ricky0123 I am getting the same issue while running VAD in react using useMicVAD.

react-dom-client.development.js:24871 Download the React DevTools for a better development experience: https://react.dev/link/react-devtools
main.jsx:8   GET http://localhost:5173/node_modules/.vite/deps/ort-wasm-simd-threaded.jsep.mjs?import net::ERR_ABORTED 404 (Not Found)
Zu @ wasm-utils-import.ts:169
Jr @ wasm-utils-import.ts:259
It @ wasm-factory.ts:154
Cu @ proxy-wrapper.ts:148
init @ backend-wasm.ts:73
Wu @ backend-impl.ts:84
hr @ backend-impl.ts:123
create @ inference-session-impl.ts:208
SileroLegacy.new @ legacy.ts:20
await in SileroLegacy.new
new @ real-time-vad.ts:237
new @ real-time-vad.ts:148
await in new
setup @ index.ts:96
(anonymous) @ index.ts:117
react_stack_bottom_frame @ react-dom-client.development.js:23953
runWithFiberInDEV @ react-dom-client.development.js:1519
commitHookEffectListMount @ react-dom-client.development.js:11905
commitHookPassiveMountEffects @ react-dom-client.development.js:12026
commitPassiveMountOnFiber @ react-dom-client.development.js:13841
recursivelyTraversePassiveMountEffects @ react-dom-client.development.js:13815
commitPassiveMountOnFiber @ react-dom-client.development.js:13957
recursivelyTraversePassiveMountEffects @ react-dom-client.development.js:13815
commitPassiveMountOnFiber @ react-dom-client.development.js:13853
flushPassiveEffects @ react-dom-client.development.js:15737
(anonymous) @ react-dom-client.development.js:15379
performWorkUntilDeadline @ scheduler.development.js:45
<App>
exports.jsxDEV @ react-jsx-dev-runtime.development.js:346
(anonymous) @ main.jsx:8
**main.jsx:8  Encountered an error while loading model file ./silero_vad_legacy.onnx**
new @ real-time-vad.ts:239
await in new
new @ real-time-vad.ts:148
await in new
setup @ index.ts:96
(anonymous) @ index.ts:117
react_stack_bottom_frame @ react-dom-client.development.js:23953
runWithFiberInDEV @ react-dom-client.development.js:1519
commitHookEffectListMount @ react-dom-client.development.js:11905
commitHookPassiveMountEffects @ react-dom-client.development.js:12026
commitPassiveMountOnFiber @ react-dom-client.development.js:13841
recursivelyTraversePassiveMountEffects @ react-dom-client.development.js:13815
commitPassiveMountOnFiber @ react-dom-client.development.js:13957
recursivelyTraversePassiveMountEffects @ react-dom-client.development.js:13815
commitPassiveMountOnFiber @ react-dom-client.development.js:13853
flushPassiveEffects @ react-dom-client.development.js:15737
(anonymous) @ react-dom-client.development.js:15379
performWorkUntilDeadline @ scheduler.development.js:45
<App>
exports.jsxDEV @ react-jsx-dev-runtime.development.js:346
(anonymous) @ main.jsx:8
main.jsx:8  Encountered an error while loading model file ./silero_vad_legacy.onnx
new @ real-time-vad.ts:239
await in new
new @ real-time-vad.ts:148
await in new
setup @ index.ts:96
(anonymous) @ index.ts:117
react_stack_bottom_frame @ react-dom-client.development.js:23953
runWithFiberInDEV @ react-dom-client.development.js:1519
commitHookEffectListMount @ react-dom-client.development.js:11905
commitHookPassiveMountEffects @ react-dom-client.development.js:12026
reconnectPassiveEffects @ react-dom-client.development.js:14004
recursivelyTraverseReconnectPassiveEffects @ react-dom-client.development.js:13976
reconnectPassiveEffects @ react-dom-client.development.js:14051
doubleInvokeEffectsOnFiber @ react-dom-client.development.js:15968
runWithFiberInDEV @ react-dom-client.development.js:1519
recursivelyTraverseAndDoubleInvokeEffectsInDEV @ react-dom-client.development.js:15928
commitDoubleInvokeEffectsInDEV @ react-dom-client.development.js:15977
flushPassiveEffects @ react-dom-client.development.js:15747
(anonymous) @ react-dom-client.development.js:15379
performWorkUntilDeadline @ scheduler.development.js:45
<App>
exports.jsxDEV @ react-jsx-dev-runtime.development.js:346
(anonymous) @ main.jsx:8

**Comment by ricky0123** [comment]
Sorry that you are having issues with the latest version. I do not have an IPhone but I did test the latest version on someone else's iphone and it is working for me. Do you have any more details that might help to debug?

**Comment by ricky0123** [comment]
All good, I'm glad it's been useful 👍

**Comment by paschaldev** [comment]
<img width="1486" height="653" alt="Image" src="https://github.com/user-attachments/assets/9c9a1b86-c369-4e57-8cd9-f53f4ccd1966" />

I had to downgrade to 0.0.32, I was getting these errors

**Comment by mayurjadhav2002** [comment]
@focux Yes, but loading these resources will take time, which is problematic for native apps.
